test/src/module/command/infoTest.c

```diff
@@ -236,16 +236,17 @@ testRun(void)
         {
             HRN_FORK_CHILD_BEGIN()
             {
-                lockInit(cfgOptionStr(cfgOptLockPath), STRDEF("999-ffffffff"), STRDEF("stanza1"), lockTypeBackup);
-                TEST_RESULT_INT_NE(lockAcquireP(), -1, "create backup/expire lock");
+                lockInit(cfgOptionStr(cfgOptLockPath), STRDEF("999-ffffffff"));
+                TEST_RESULT_INT(
+                    lockAcquireP(cmdLockFileName(STRDEF("stanza1"), lockTypeBackup)), true, "create backup/expire lock");
```

Comment:
Is there a reason we compare lockAcquireP's result to INT in infoTest.c vs BOOL in the other test files?

Reply:
No reason -- just incautious search and replace. Fixed in 95f3baad267.